cuML/src/spectral/spectral.h

```diff
@@ -0,0 +1,258 @@
+#pragma once
+
+#include <nvgraph.h>
+
+#include "sparse/coo.h"
+#include "knn/knn.h"
+
+#include "cuda_utils.h"
+
+namespace ML {
+
+    namespace Spectral {
+
+        #define NVGRAPH_CHECK(call)  \
```

Comment:
This guy probably should go in a `nvgraph_wrappers.h` file for consistency with current prims